src/classes/syncer/WebResourceSyncerConfigurationManager.ts

```diff
@@ -60,4 +69,36 @@ export abstract class WebResourceSyncerConfigurationManager {
 			throw new Error(`No property named ${propertyName} in package.json`);
 		}
 	}
+
+	public static async saveWebResourceFileMapping(localPath: string, remotePath: string): Promise<void> {
+		let config;
+		try {
+			config = await this.getConfigFileAsJson();
+		}
+		catch {
+			return;
```

Comment:
Should we do something here? I left this empty since I don't know what the vscode API offers.